### PR TITLE
adding known issues

### PR DESCRIPTION
**Pull request by linsun**
https://github.com/istio/istio.github.io/issues/643

### CODE REVIEW

File: _docs/reference/release-notes.md [hunk 1]
```diff
@@ -88,5 +88,10 @@ identity provisioning. This agent runs on each node (VM / physical machine) and
 - **Persistent CA Key/Certificate Storage**. Istio CA now stores signing key/certificates in
 persistent storage to facilitate CA restarts. 
 
+### Known Issues
+
+- **User may get periodical 404 when accessing the application**:  We have noticed that envoy doesn't get routes properly occasionally thus a 404 is returned to the user.  We are actively working on this [issue](https://github.com/istio/istio/issues/1038).
+- **Pilot reports ready before Pilot is actually ready**: You can check pods status in the `istio-system` namespace and wait a few seconds after all Istio Pods reach ready status.  We are actively working on this [issue](https://github.com/istio/istio/pull/1055).
```
Comment (ldemailly) [inline _docs/reference/release-notes.md]: replace Pilot by Istio-Ingress ?

File: _docs/reference/release-notes.md [hunk 2]
```diff
@@ -88,5 +88,10 @@ identity provisioning. This agent runs on each node (VM / physical machine) and
 - **Persistent CA Key/Certificate Storage**. Istio CA now stores signing key/certificates in
 persistent storage to facilitate CA restarts. 
 
+### Known Issues
```
Comment (ldemailly) [inline _docs/reference/release-notes.md]: maybe we should also list here that with auth enabled you can't talk to k8s api server or non istio services
(and that for health checks you need to use a custom curl command)

File: _docs/reference/release-notes.md [hunk 3]
```diff
@@ -88,5 +88,10 @@ identity provisioning. This agent runs on each node (VM / physical machine) and
...
+
+- **User may get periodical 404 when accessing the application**:  We have noticed that envoy doesn't get routes properly occasionally thus a 404 is returned to the user.  We are actively working on this [issue](https://github.com/istio/istio/issues/1038).
```
Comment (geeknoid) [inline _docs/reference/release-notes.md]: envoy -> Envoy

File: _docs/reference/release-notes.md (hunk 1 above)
Comment (geeknoid) [inline _docs/reference/release-notes.md]: pods status -> pod status
Istio Pods -> Istio pods
ready status -> ready state

File: _docs/reference/release-notes.md (hunk 3 above)
Comment (linsun) [reply _docs/reference/release-notes.md]: Thanks.

File: _docs/reference/release-notes.md (hunk 2 above)
Comment (linsun) [reply _docs/reference/release-notes.md]: these are already documented in security FAQ, but I did add non istio service as that is pretty major.  talking to K8S API server isn't that common...

File: _docs/reference/release-notes.md (hunk 1 above)
Comment (linsun) [reply _docs/reference/release-notes.md]: updated.

Comment (linsun) [reply _docs/reference/release-notes.md]: updated.  I kept status as that is what kubectl output shows when I call `kubectl get pods`

**Pull request merged**